scripts/upload-file-to-s3.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+"""
+Usage:
+    scripts/upload-file-to-s3.py
+        <stage> <file_path> <bucket_category> [<remote_path>] [<remote_name>] [--dry-run] [--public]
+
+Options:
+    -h --help   Show this screen.
+    --stage=<stage>  The stage being updated
+    --file_path=<file_path>  Path of file to be saved in S3
+    --bucket_category=<bucket_category>  Bucket category, used to get name ie agreements/ communications/ documents
+    --remote_path=<remote_path>  Path to save the file at inside the bucket
+    --download_name=<download_name>  Suggested name for a browser to download, part of Content-Disposition header
```

Comment:
Can you give an example of this being used? Not sure I understand it.

Reply:
Hey so this sets the content-disposition header attribute of the file which determines whether it will be opened in browser or downloaded as a file.
https://github.com/alphagov/digitalmarketplace-utils/blob/a33fad66d50bc0e9f06193cac28c015a4a292121/dmutils/s3.py#L53-L53

Reply:
Ah ok, so it's just a header to tell a browser what to do with the thing it's getting?